estimate_length.py

```diff
@@ -205,22 +205,26 @@ def extract_pAi_from_genome(genome, window, occurences, consecutive):
             while c <= len(line)-window:
                 segment = line[c:(c+window)]
                 if consecutive*'A' in segment:
```

Comment:
not sure if I get the sytax here: `consecutive*'A' in segment` is `consecutive * ('A' in segment)` which is `consecutive and ('A' in segment)`? If so, I'd prefer the latter for clarity unless it has some drawbacks I'm not aware of.